### DISCUSSION TITLE
Daily 3/11

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#270] [General]
Ayer limpié un poco más el código de Python, expandí algo la sección del README, tal vez termine creando un archivo con la documentación de desarrollo en Documentación para que no quede tan larga (como ejecutar, como testear, etc). También me puse a actualizar el modelo de clases y quedó bastante más grande que antes, pasé todas las clases del modelo incluyendo las de memoria y base de datos, hoy tendría que pasar las del servicio y los controladores pero creo que voy a hacerlo en otro diagrama para que no quede tan complicado. Hoy veo la clase de SGC y me pongo con lo de mi CV y también empezar a ver lo del sitio de Almoneda. También quiero estar en la clase de full-stack.

**Comment by NataliaAlvarezIspc** [comment]
Ayer terminé de diseñar el footer, de copiarlo en todas las páginas y de subirlo al repositorio. También empecé a armar algo para el video.
Hoy continuaré armando lo que voy a decir en el video, aunque no tengo muy en claro que decir. Voy a leer los contenidos de SGC para la clase de hoy. Por la tarde-noche participaré de las clases de SGC y Full
